Prac_06/guitar_test.py

```diff
@@ -0,0 +1,8 @@
+from Prac_06.guitar import Guitar
```

Comment:
This code doesn't seem to be finished, maybe you can check the solutions at:[https://github.com/CP1404/Practicals/tree/solutions/prac_06](url)
:)